cli/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/supitsdu/clipper/cli/clipper"
+	"github.com/supitsdu/clipper/cli/options"
+)
+
+func main() {
+	config := options.ParseFlags()
+	writer := clipper.DefaultClipboardWriter{} 
+	clipper.Run(config, writer)

```

Comment:
```suggestion
err :=	clipper.Run(config, writer)
if err != nil { 
                 fmt.Printf("Error %s", err) 
                 os.Exit(1)
}
```

Only main should use os.Exit